### PR TITLE
[issue-36] integrate tools-python

### LINKED ISSUE #36: Integrate SPDX Python tools
https://github.com/spdx/tools-python

### PR DESCRIPTION
**Pull request by meretp**
fixes #36 

Task 7 fails and is therefore not included in the workflow, I opened an issue for that: #50.

Signed-off-by: Meret Behrens <meret.behrens@tngtech.com>

### CODE REVIEW

**Comment by meretp** [comment]
As soon as #53 is merged this PR can also be reviewed. #53 adds the missing artifacts so that the workflow should no longer fail once ihe PR is merged.

File: .github/workflows/tools-python.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: TNG Technology Consulting GmbH
```
Comment (armintaenzertng) [inline .github/workflows/tools-python.yaml:1]: Is this a thing that we want to keep?

**Review by armintaenzertng — COMMENTED** [review]
Looks good, I only have a few minor remarks.

File: spdx-tools-python-solver/testcase_generationBaselineSbom.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+from spdx.writer.xml.xml_writer import write_document_to_file
+
+
+@click.command()
+@click.option("-target", "-t", help="Path where the generated file will be stored.")
+def main(target: str):
+    creation_info = CreationInfo(spdx_version="SPDX-2.3", spdx_id="SPDXRef-DOCUMENT", name="document name",
+                                 document_namespace="https://some.namespace", data_license="CC0-1.0",
+                                 creators=[Actor(ActorType.TOOL, "test-tool")],
+                                 created=datetime(2022, 1, 1, 0, 0))
```
[the remark this answers is not in the document]
Comment (meretp) [reply spdx-tools-python-solver/testcase_generationBaselineSbom.py:35]: fixed

File: .github/workflows/tools-python.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+#
+# SPDX-License-Identifier: Apache-2.0
+name: "Testcases for tools-python"
+
+on:
+ [ workflow_dispatch, pull_request]
```
[the remark this answers is not in the document]
Comment (meretp) [reply .github/workflows/tools-python.yaml:7]: Agreed

File: .github/workflows/tools-python.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+jobs:
+  run-testcases:
+    name: "Run Testcases"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```
[the remark this answers is not in the document]
Comment (meretp) [reply .github/workflows/tools-python.yaml:15]: Yes, this is necessary to be able to run the python scripts for the testcases. Makes perfectly sense for me as the scripts are part of the testbed repo, so we need to check the code out.

File: .github/workflows/tools-python.yaml (hunk 1 above)
Comment (meretp) [reply .github/workflows/tools-python.yaml]: No, I changed it to `SPDX Contributors` and also updated the license header in the python files.

File: .github/workflows/tools-python.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (armintaenzertng) [reply .github/workflows/tools-python.yaml:15]: Ah of course, you are right

**Review by armintaenzertng — APPROVED** [review]
Thanks for the fixes :)

**Pull request merged**